package.json

```diff
@@ -47,7 +47,8 @@
   "scripts": {
     "start": "PORT=3002 react-scripts start",
     "server": "npm run start:watch --prefix ../backend/",
-    "renderer": "(cd ../renderer-docker && ./start_docker.sh)",
+    "renderer": "(cd ../renderer && sudo morbo ./script/render_app)",
```

Comment:
I don't like this, there are assumptions about your system, I would rather this was in a parent project with submodules